plugin.rb

```diff
@@ -26,10 +26,22 @@ module ::DiscourseCustomTopicLists
         []
       end
     current_user = scope.user
-    custom_lists.select do |list|
-      allowed_groups = list["access"].split(/(?:,|\s)\s*/).map(&:to_i)
-      allowed_groups = [Group::AUTO_GROUPS[:everyone]] if allowed_groups.empty?
-      current_user.in_any_groups?(allowed_groups)
+    raw_custom_lists =
+      custom_lists.select do |list|
+        allowed_groups = list["access"].split(/(?:,|\s)\s*/).map(&:to_i)
+        allowed_groups = [Group::AUTO_GROUPS[:everyone]] if allowed_groups.empty?
+        current_user.in_any_groups?(allowed_groups)
+      end
+    raw_custom_lists.map do |list|
+      markdown =
+        Redcarpet::Markdown.new(
+          Redcarpet::Render::HTML.new(hard_wrap: true),
+          no_intra_emphasis: true,
+          fenced_code_blocks: true,
+          autolink: true,
+        )
+      list["description"] = markdown.render(list["description"])
```

Comment:
There's no need for any external gems. I'm sure what you need is our md rendering pipeline:

```suggestion
      list["description"] = PrettyText.cook(list["description"])
```

Reply:
Thanks a lot!! I was trying to find a way to use it on core. 


I'll remove references of the `redcarpet`